src/style.css

```
@@ -1,8 +1,16 @@
+html, body, #app {
+  transition: background-color .35s;
+}
+
 body {
   margin: 20px;
   background-color: #f2f2f2;
 }
 
+.dark body {
+  background-color: #221212;
```

```suggestion
  background-color: #121212;
```
Looks a bit red, you may mean: `#121212`?